src/wmnds/patterns/header-v2/_header-v2.njk

```diff
@@ -0,0 +1,153 @@
+{% from "wmnds/patterns/banner/_banner.njk" import wmndsBanner %}
+{% from "wmnds/components/breadcrumb/_breadcrumb.njk" import wmndsBreadcrumb %}
+
+<!-- Skip to content link -->
+{{
+  wmndsLink({
+    link: '#wmnds-main-content',
+    linkTitle: 'Skip to main content',
+    text: 'Skip to main content',
+    classModifiers: 'wmnds-header__skip-link'
+  }) | trim
+}}
+<!-- Main header section -->
+<header>
+  <div class="wmnds-header{{appClass}} wmnds-header--mega-menu">
+    <div class="wmnds-container wmnds-grid wmnds-grid--align-center wmnds-grid--justify-between">
+      <div class="wmnds-header__vertical-align wmnds-col-auto">
+        <!-- Logo -->
+        <a class="wmnds-header__logo-link" href="/" title="West Midlands Network Design System">
+          <img class="wmnds-header__logo" alt="West Midlands Network logo" src="/img/logo.svg" />
+        </a>
+      </div>
+      {#- If with a title #}
+      {% if params.withTitle -%}
+      <!-- Header title -->
+      <h1 class="wmnds-header__title wmnds-col-1 wmnds-col-sm-auto">
+        Reinstate your Direct Debit<br>(COVID-19)
+      </h1>
+      {%- else -%}
+      <!-- Mega menu nav items-->
+      <nav id="{{params.id}}" class="wmnds-mega-menu">
+        <!-- Mobile toggle button -->
+        <button class="wmnds-mega-menu__mobile-toggle wmnds-btn wmnds-btn--secondary wmnds-hide-desktop" aria-expanded="false" aria-controls="{{params.id}}-primary-menu">Menu
+          <svg class="wmnds-mega-menu__close-icon"><use xlink:href="#wmnds-general-cross" href="#wmnds-general-cross"></use></svg>
+        </button>
+        <!-- Container for mega menu - allows scrolling on mobile nav -->
+        <div id="{{params.id}}-primary-menu" class="wmnds-mega-menu__scroll-controller">
+          <!-- Start primary (level 1) navigation -->
+          <ul class="wmnds-mega-menu__primary-menu">
+          {% for navItem in navItems -%}
+            <li class="wmnds-mega-menu__primary-menu-item">
+            <!-- Show swift logo in nav if is swift & tickets link -->
+            {% set navItemIcon = wmndsIcon({icon: 'swift-full-logo', class: 'wmnds-swift-icon'}) if navItem.name === "Swift and tickets" else "" -%}
+            {% set navItemContent = " and tickets" if navItem.name === "Swift and tickets" else navItem.name -%}
+              <!-- allow primary (level 1) item to link if specified -->
+              {% if params.linkTopLevel -%}{{navItemIcon}}
+              <a href="/{{ navItem.name | lower | replace(" ", "-") }}/" title="{{ navItem.name }} page" target="_self" class="wmnds-mega-menu__primary-menu-link" >
+                {{navItemIcon}}{{ navItemContent }}
+              </a>
+              <!-- if not linked use a button -->
+              {% else -%}
+              <button target="_self" class="wmnds-mega-menu__primary-menu-link" >
+                {{navItemIcon}}{{ navItemContent }}
+              </button>
+              {% endif -%}
```

Comment:
Indentation is off here, I think the `wmndsIcon()` call on line 52 just needs an indent

![image](https://user-images.githubusercontent.com/43111519/92707203-03341e00-f34d-11ea-952b-1c5f28506a73.png)